script/ovm/README.md

```diff
@@ -0,0 +1,204 @@
+# ObolValidatorManager scripts
+
+## RPC Endpoints
+
+You can use any RPC endpoint, but here are some public ones suitable for running these scripts:
+
+- Mainnet: https://ethereum-rpc.publicnode.com
+- Sepolia: https://ethereum-sepolia-rpc.publicnode.com
+- Holesky: https://ethereum-holesky-rpc.publicnode.com
+- Hoodi:   https://ethereum-hoodi-rpc.publicnode.com
+
+Note: all the scripts and examples below are designed to run on the Holesky network.
```

Comment:
Shall we make them for Hoodi?

Reply:
Made Sepolia being the default network, as discussed internally.